src/autoupdater.ts

```diff
@@ -107,7 +114,7 @@ class AutoUpdater {
     return true;
   }
 
-  async prNeedsUpdate(pull) {
+  async prNeedsUpdate(pull: any): Promise<boolean> {
```

Comment:
I thought that this type should have been `PullsListResponseItem` like for `update()`, but using it gives the error `Property 'merged' does not exist on type 'PullsListResponseItem'`

Reply:
Maybe a type out of https://github.com/octokit/webhooks is what you're looking for? This isn't sent via an API request, GH actions calls autoupdate with an pull event